includes/embeds/class-amp-youtube-embed-handler.php

```diff
@@ -77,6 +79,76 @@ public function unregister_embed() {
 		remove_filter( 'embed_oembed_html', [ $this, 'filter_embed_oembed_html' ], 10 );
 	}
 
+	/**
+	 * Sanitize HTML that are not added via Gutenberg.
```

Comment:
Hey @pierlon 

I have tried removing `filter_embed_oembed_html()`, But after that, during the `the_content` filter it removes extra query parameters from the youtube URL. Like
From 
```
http://www.youtube.com/watch?v=kfVsfOSbJY0&hl=en&fs=1&w=425&h=349
```
To 
```
<p><iframe title="Rebecca Black - Friday" width="980" height="551" src="https://www.youtube.com/embed/kfVsfOSbJY0?feature=oembed" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></p>
```

Because of that additional param will be removed and won't be able to set in `<amp-youtube/>` component during the `::sanitize_raw_embeds()`